examples/todoapp/lib/base/repositories/deep_link_repository.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import '../common_mappers/error_mappers/error_mapper.dart';
+import '../data_sources/remote/deep_link_remote_data_source.dart';
+import '../models/deep_link_model.dart';
+import '../models/response_models/deep_link_list_response_model.dart';
+
+class DeepLinkRepository {
```

Comment:
As agreed all generated by the rx_bloc_cli files, which are not used need to be removed

Reply:
Done